firmware/klipper_configurations/Octopus/Voron2_Octopus_Config.cfg

```diff
@@ -0,0 +1,646 @@
+## Voron Design VORON2 250/300/350mm BTT Octopus TMC2209 UART config
+
+## *** THINGS TO CHANGE/CHECK: ***
+## MCU paths							[mcu] section
+## Thermistor types						[extruder] and [heater_bed] sections - See 'sensor types' list at end of file
+## Z Endstop Switch location			[safe_z_home] section
+## Homing end position				[gcode_macro G32] section
+## Z Endstop Switch  offset for Z0		[stepper_z] section
+## Probe points							[quad_gantry_level] section
+## Min & Max gantry corner postions		[quad_gantry_level] section
+## PID tune								[extruder] and [heater_bed] sections
+## Fine tune E steps					[extruder] section
+
+##========================== Pin Definitions ========================
+## M0_STEP_PIN         PF13
+## M0_DIR_PIN          PF12
+## M0_ENABLE_PIN       PF14
+## M0_UART             PC4
+
+## M1_STEP_PIN         PG0
+## M1_DIR_PIN          PG1
+## M1_ENABLE_PIN       PF15
+## M1_UART             PD11
+
+## M2_STEP_PIN         PF11
+## M2_DIR_PIN          PG3
+## M2_ENABLE_PIN       PG5
+## M2_UART             PC6
+
+## M3_STEP_PIN         PG4
+## M3_DIR_PIN          PC1
+## M3_ENABLE_PIN       PA0
+## M3_UART             PC7
+
+## M4_STEP_PIN         PF9
+## M4_DIR_PIN          PF10
+## M4_ENABLE_PIN       PG2
+## M4_UART             PF2
+
+## M5_STEP_PIN         PC13
+## M5_DIR_PIN          PF0
+## M5_ENABLE_PIN       PF1
+## M5_UART             PE4
+
+## M6_STEP_PIN         PE2
+## M6_DIR_PIN          PE3
+## M6_ENABLE_PIN       PD4
+## M6_UART             PE1
+
+## M7_STEP_PIN         PE6
+## M7_DIR_PIN          PA14
+## M7_ENABLE_PIN       PE0
+## M7_UART             PD3
+
+## Endstop Pins
+## DIAG_0              PG6
+## DIAG_1              PG9
+## DIAG_2              PG10
+## DIAG_3              PG11
+## DIAG_4              PG12
+## DIAG_5              PG13
+## DIAG_6              PG14
+## DIAG_7              PG15
+
+## Fan Pins
+## FAN0                PA8
+## FAN1                PE5
+## FAN2                PD12
+## FAN3                PD13
+## FAN4                PD14
+## FAN5                PD15
+
+## Thermistor Pins
+## TB                  PF3
+## T0                  PF4
+## T1                  PF5
+## T2                  PF6
+## T3                  PF7
+
+## Heater Pins
+## BED_OUT             PA1
+## HE0                 PA2
+## HE1                 PA3
+## HE2                 PB10
+## HE3                 PB11
+
+## MISC Pins
+## SENSOR              PB7 (make sure to set jumper correctly)
+##===================================================================
+
+[mcu]
+##	Obtain definition by "ls -l /dev/serial/by-id/" then unplug to verify
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/{REPLACE WITH YOUR SERIAL}
+##--------------------------------------------------------------------
```

Comment:
need to add `restart_method: command` the the [mcu] section for resets to work properly on some boards.